docs/nlu/components.rst

```diff
@@ -451,6 +451,9 @@ EmbeddingIntentClassifier
               ``inner`` for ``softmax``, ``cosine`` for ``margin``;
             - ``loss_type`` sets the type of the loss function,
               it should be either ``softmax`` or ``margin``;
+            - ``ranking_length`` defines the number of top confidences over
+              which to normalize ranking results if ``loss_type: "softmax"``;
+              to turn off normalization set to 0
```

Comment:
do we have similar docs for response selector?

Reply:
No, there is no list of parameters. but it shows the defaults which have a comment about it in. I could write a note?

Reply:
Do you know how response selector interacts with fallback policy? if there is actually a relevant threshold (core probably?) for which the normalization will influence success, i will add a note. if no threshold i think the list of defaults is fine since it won't change behavior

Reply:
I  response selector + fallback is still an open issue

Reply:
ok, i think this is fine for now then